test/functional/runner/index.js

```diff
@@ -105,7 +105,13 @@ function shouldSkipTest(spec) {
     'commitTransaction retry succeeds on new mongos',
     'commitTransaction retry fails on new mongos',
     'unpin after transient error within a transaction and commit',
-    'count'
+    'count',
+
+    // we do update validation before the expectation in this test
+    'an invalid update \(no \$ operators\) is validated and errors', // eslint-disable-line
```

Comment:
Nit: for the future, we should make a `skipList` that we pass in to individual calls to `generateTopologyTests` instead of having a master list here.